Library/Homebrew/shims/linux/super/cc

```diff
@@ -184,6 +184,11 @@ class Cmd
       val  = chuzzle($1) || enum.next
       path = canonical_path(val)
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
+    when /^-R(.+)?/
+      # -R is not the correct way to pass rpaths to the linker
+      val  = chuzzle($1) || enum.next
+      path = canonical_path(val)
+      args << "-Wl,-R," if keep?(path) && @lset.add?(path)
```

Comment:
> `&& @lset.add?(path)`

This doesn't seem right. If it's already in `lset` because it was added by `-L`, then the `-rpath` option won't be added as expected.